index.js

```diff
@@ -0,0 +1,175 @@
+let express = require('express');
+let cors = require('cors');
+let bodyParser = require('body-parser');
+let app = express();
+
+// This is necessary middleware to parse JSON into the incoming request body for POST requests
+app.use(bodyParser.json());
+
+/**
+ * Security Considerations:
+ * - CDS Services must implement CORS in order to be called from a web browser:
+ * - See details here: http://cds-hooks.org/#security
```

Comment:
Do we think the security PR will land before CHC? (Probably not) This doesn't match up with what is documented. We could link to the PR https://cds-hooks-docs-pr-72.herokuapp.com/#security, but that will get out of date eventually.

Reply:
The CORS part is already in the spec; it's the only thing under security.
http://cds-hooks.org/#cross-origin-resource-sharing

Reply:
But the headers different.
<img width="708" alt="screen shot 2017-09-29 at 1 44 02 pm" src="https://user-images.githubusercontent.com/1588747/31030768-2adcd780-a51c-11e7-8be4-a14457389dca.png">

Reply:
Ah I see what you mean. We can keep the code there though, just remove the spec url reference until that security PR gets merged in?